src/arrayMethodJoin.js

```diff
@@ -4,8 +4,26 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    const join2 = [...this];
+    let result = '';
+
+    for (let i = 0; i < join2.length; i++) {
+      if (join2[i] === undefined || join2[i] === null) {
+        join2[i] = '';
+      }
```

Comment:
What's the purpose of changing the array elements? Just add an element to the result if it's not undefined or null